tests/client/test_stdio_cm.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from mcp import StdioServerParameters
+from mcp.client.stdio import stdio_client
+
+
+@pytest.mark.anyio
+async def test_context_manager_exiting():
+    async with stdio_client(StdioServerParameters(command="tee")) as (
```

Comment:
When moved to `test_stdio` tee is already declared

```suggestion
    async with stdio_client(StdioServerParameters(command=tee)) as (
```